data/services.json

```diff
@@ -282,6 +282,22 @@
         "color": "#00ec82d9",
         "website": "https://github.com/onflow/fcl-dev-wallet"
       }
+    },
+    {
+      "f_type": "Service",
+      "f_vsn": "1.4.1",
+      "type": "authn",
+      "method": "HTTP/POST",
+      "uid": "devwallethttppost#authnpost",
+      "endpoint": "http://localhost:8701/api/authn",
```

Comment:
apart from `f_vsn`, `method`, `uid`, and `endpoint` everything else is a copy of a local `IFRAME/RPC` method above